Customizer/Customizer.py

```diff
@@ -67,7 +67,12 @@ def showCustomizerWelcome(self):
 
   def hideModules(self):
     modules = [@MODULES_TO_HIDE@]
-    mm = slicer.util.findChildren(className='qSlicerModulesMenu')[0]
+    try:
+      mm = slicer.util.findChildren(className='qSlicerModulesMenu')[0]
+    except:
```

Comment:
It's suggested form to always specify which exception you expect (i.e. AttributeError or the like).  

Also the issue here looks like some kind of bug in findChildren.  That class has recently had some bug fixing and edits in the slicer master branch that change the way it works.  Maybe those changes have an impact on the behavior here?  It would be better to avoid this workaround or flag that it's an issue with 4.5 but fixed in subsequent versions of slicer.